src/backend/InvenTree/report/templatetags/report.py

```diff
@@ -396,42 +398,51 @@ def render_html_text(text: str, **kwargs):
 
 
 @register.simple_tag
-def format_number(number, **kwargs):
+def format_number(
+    number,
+    decimal_places: Optional[int] = None,
+    integer: bool = False,
+    leading: int = 0,
+    separator: Optional[str] = None,
+) -> str:
     """Render a number with optional formatting options.
 
-    kwargs:
+    Arguments:
         decimal_places: Number of decimal places to render
         integer: Boolean, whether to render the number as an integer
-        leading: Number of leading zeros
+        leading: Number of leading zeros (default = 0)
+        separator: Character to use as a thousands separator (default = None)
     """
     try:
         number = Decimal(str(number))
     except Exception:
         # If the number cannot be converted to a Decimal, just return the original value
         return str(number)
 
-    if kwargs.get('integer', False):
+    if integer:
         # Convert to integer
         number = Decimal(int(number))
 
     # Normalize the number (remove trailing zeroes)
     number = number.normalize()
 
-    decimals = kwargs.get('decimal_places')
+    decimal_places
```

Comment:
@SchrodingersGat was this intended to be left in here? Should I PR a removal?

Reply:
@matmair good find. Fix incoming in https://github.com/inventree/InvenTree/pull/8529

Reply:
Thanks, this was thrown by sonarcloud